third_party/unix/zerrors_linux.go

```diff
@@ -0,0 +1,34 @@
+// Code generated by mkmerge; DO NOT EDIT.
+
+//go:build linux
+// +build linux
+
+/*
+// Copyright 2021 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+Modifies:
+- All the consts defined below are copied from the original code.
```

Comment:
The geneted files has both "DO NOT EDIT" and "Modifies", have they been updated manually?
Asking for the maintain of such code, could you make README more complete to list what files are written by us and what files are gererated, and the whole steps to generate the code. I think Jianjun has this concern too: https://github.com/antrea-io/antrea/pull/2835#discussion_r782488896

Reply:
README updated. Modifies are added by myself. compile command and DO NOT EDIT are copied from the original file, for instance,  https://github.com/golang/sys/blob/5a964db013201115fcba5c3d31ade965d0969335/unix/ztypes_linux_amd64.go#L2

Reply:
So we do update them? The "DO NOT EDIT" warning will confuse people. Since you already remove many code, why don't remove the confusing comment which is false here.
Better to have a link of the original code.
You didn't link mention your update in this file in README, could you add it and anything missing if there is?

Reply:
`zerrors_linux.go` has already removed